editor/src/core/model/conditionals.ts

```diff
@@ -330,3 +331,43 @@ export function findFirstNonConditionalAncestor(
   }
   return findFirstNonConditionalAncestor(parent, metadata)
 }
+
+export function isTextEditableConditional(
+  path: ElementPath,
+  metadata: ElementInstanceMetadataMap,
+  elementPathTree: ElementPathTrees,
+): boolean {
+  const element = MetadataUtils.findElementByElementPath(metadata, path)
+  if (element == null) {
+    // element doesn't exist
+    return false
+  }
+  const conditional = maybeConditionalExpression(element)
+  if (conditional == null) {
+    // element is not a conditional
+    return false
+  }
+  const nonConditionalAncestor = findFirstNonConditionalAncestor(path, metadata)
+  const siblings = MetadataUtils.getChildrenOrdered(
+    metadata,
+    elementPathTree,
+    nonConditionalAncestor,
+  )
+  if (siblings.length > 1) {
```

Comment:
should this be a safer `!== 1` ?

Reply:
For me this is more readable, because it expresses that the problem is when there are real siblings (one "sibling" is the conditional itself, so 0 is not possible).